exercises/exercise3.js

```diff
@@ -8,5 +8,145 @@ module.exports.botScripts = [
     handler: function () {
       return 'Good Evening to You!';
     },
+    //I want the bot to find the to be have a varity of moods when asked a question.
+
+    label: 'Whats your mood bot',
+    prompt: 'Next are you happy',
+    handler: function () {
+      return 'Yes i am';
+    }, //this will tell me what my name is
+    label: 'I want to make a discord bot',
+    prompt: 'What is my #{name}',
+    handler: function (params) {
+      const customer = params.name;
+      return 'Your name is ' + params.name;
+    },
+    isCaseSensitive: false,
+    isListening: true,
+  },
```

Comment:
Technically, it looks like the second and third functions are in the same {} as your example script. This can and most likely is causing some problems. Adding back that missing closing curly brace on your first interaction and the missing opening curly brace for the second interaction will help. Then it's a matter of moving these previous two functions to the left by one tab (shift + tab).